kubernetes/zenko/charts/cloudserver/templates/manager-deployment.yaml

```diff
@@ -4,15 +4,15 @@ kind: Deployment
 metadata:
   name: {{ template "cloudserver.fullname" . }}-manager
   labels:
-    app: {{ template "cloudserver.name" . }}
```

Comment:
I think we should start using partials for the different component names that a chart can have.